scolapasta-int-parse/src/radix.rs

```diff
@@ -0,0 +1,252 @@
+use core::num::NonZeroU32;
+
+// Create a lookup table from each byte value (of which the ASCII range is
+// relevant) to the maximum minimum radix the character is valid for.
+#[allow(clippy::cast_possible_truncation)]
```

Comment:
This lint can be removed if the loop counter is a `u8` and the as cast widens to `usize`

Reply:
done